gtfs2ntfs/src/main.rs

```diff
@@ -58,6 +58,19 @@ struct Opt {
     #[structopt(long = "odt-comment")]
     odt_comment: Option<String>,
 
+    /// If we don’t have the flag activated,
+    ///     2 GTFS routes with the same route_short_name
+    ///         they have the same agency_id: the 2 routes are merged in the same line
+    ///         they have different agency_id: we will have 2 lines in output
+    ///     2 GTFS routes with an empty route_short_name and the same route_long_name
+    ///         they have the same agency_id: the 2 routes are merged in the same line
+    ///         they have different agency_id: we will have 2 lines in output
+    /// If the flag is activated:
+    ///     2 GTFS routes with the same route_short_name (and even agency_id) will generate 2 lines
+    ///     2 GTFS routes with the same route_long_name (and even agency_id) will generate 2 lines
+    #[structopt(long = "read-as-line")]
+    read_as_line: bool,
```

Comment:
wouldn't it be better to explicitly set the default value to false in structopt too?

Reply:
default value is meaningless for bool

Reply:
oh right I forgot it was a flag sorry